src/pages/projects/actions/DeleteProjectBtn.tsx

```diff
@@ -30,12 +32,12 @@ const DeleteProjectBtn: FC<Props> = ({ project }) => {
   const isDefaultProject = project.name === "default";
   const isEmpty = isProjectEmpty(project);
   const getHoverText = () => {
+    if (!isEmpty) {
+      return "";
+    }
     if (isDefaultProject) {
       return "The default project cannot be deleted";
     }
```

Comment:
The default project - which _is_ used by several things - displays a the message "Non-empty project cannot be deleted." rather than this message. Perhaps rather than a display either message X or message Y, this warning ("The default project cannot be deleted") can be conditionally appended later on.

Reply:
I think appending the default project message might make the tooltip popup too busy and crowded. Since a default project can never be deleted, I think we can just show the message "The default project cannot be delete"?

Reply:
Can do, but the problem I am highlighting is the fact that according to the code logic, two different messages _should_ be showing up, but only one of them is. We could either merge them, which may cause crowding, as you mentioned, or just show one of them.

Reply:
The current logic will show the usedBy tooltip if the project is not empty, even if the project is the default project. The new approach will show "The default project cannot be delete" for the default project, and the usedBy tooltip for non-default projects that are not empty.